### PR TITLE
Lab 1

### PR DESCRIPTION
**Pull request by edr3x**
Lab 1 complete

### CODE REVIEW

File: app-script-payslip-sender/email_sender.gs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  let secondLastColumn = sheet.getLastColumn() - 1;
+
+  let range = sheet.getRange(2, 2, lastRow - 1, secondLastColumn);
+
+  let values = range.getValues();
+
+  for (let i = 0; i < values.length; i++) {
+    let status;
+    let [Name, Email, Salary] = values[i];
```
Comment (ritushishir) [inline app-script-payslip-sender/email_sender.gs]: better to follow the standard naming convention for variables in JavaScript.

https://www.30secondsofcode.org/js/s/naming-conventions/

File: app-script-payslip-sender/email_sender.gs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    if (!Email) {
+      status = "No Email Provided";
+    } else {
+      try {
+        let msg = buildMessage(Name, Salary);
+        MailApp.sendEmail(Email, "Salary for Month of May", msg);
+        status = "Success";
+      } catch (err) {
+        console.log(err);
```
Comment (ritushishir) [inline app-script-payslip-sender/email_sender.gs:27]: does it execute without an error?

**Review by ritushishir — CHANGES_REQUESTED** [review]

File: app-script-email-for-form-submission/thank_you_email.gs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+function sendThankYouEmail() {
+  let responses = FormApp.getActiveForm().getResponses();
+
+  let email = responses.pop().getItemResponses()[1].getResponse();
+
+  let msg = "Thank you for submitting the form.";
+
+  if (isValidEmail(email)) {
```
Comment (ritushishir) [inline app-script-email-for-form-submission/thank_you_email.gs:8]: 👍

**Comment by edr3x** [comment]
all requested changes applied!

File: app-script-payslip-sender/email_sender.gs (hunk 2 above)
Comment (ritushishir) [reply app-script-payslip-sender/email_sender.gs:27]: This line raises another error in App Script. Can you try with few inputs that triggers this line?

**Review by ritushishir — APPROVED** [review]

**Pull request merged**